### PR TITLE
ZilWall space

### PR DESCRIPTION
**Pull request by x0rb0t**
**Add space for ZilWall**  

Website: https://zilwall.com/
Twitter: https://twitter.com/zwall_zpaint
Telegram: https://t.me/zilwall_community

### CODE REVIEW

File: skins/zwall.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.zwap {
```
Comment (hicaru) [inline skins/zwall.scss]: Should `zwall` selector

File: skins/zwall.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  --text-color: #101832;
+  --link-color: #6F16FD;
+  --heading-color: #101832;
+  --border-color: #6F16FD;
+  --header-bg: #ffffff;
+  --block-bg: #ffffff;
+}
+
+@media (prefers-color-scheme: dark) {
+  .port {
```
Comment (hicaru) [inline skins/zwall.scss]: Should be `zwall` selector

File: skins/zwall.scss (hunk 1 above)
Comment (x0rb0t) [reply skins/zwall.scss]: https://github.com/Zilliqa/snapshot-spaces/pull/30/commits/7d5afe4aef105a072fd4b65dea606305a043b26d

File: skins/zwall.scss (hunk 2 above)
Comment (x0rb0t) [reply skins/zwall.scss]: https://github.com/Zilliqa/snapshot-spaces/pull/30/commits/7d5afe4aef105a072fd4b65dea606305a043b26d

**Review by hicaru — APPROVED** [review]

**Pull request merged**